pipe-cli/src/config.py

```diff
@@ -12,33 +12,133 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import click
+import base64
+from functools import update_wrapper
 import json
 import os
 import pytz
 import tzlocal
+from datetime import datetime, timedelta
 from pypac import api as PacAPI
 from pypac.resolver import ProxyResolver as PacProxyResolver
 
 from .utilities import time_zone_param_type
 
 PROXY_TYPE_PAC = "pac"
 PROXY_PAC_DEFAULT_URL = "https://google.com"
+ALL_ERRORS = Exception
+
 
 class ConfigNotFoundError(Exception):
     def __init__(self):
         super(ConfigNotFoundError, self).__init__('Unable to locate configuration or it is incomplete. '
                                                   'You can configure pipe by running "pipe configure"')
 
 
+def check_token(token, timezone, print_info=False):
+    if not token:
+        click.echo(click.style('No access token is provided', fg='red'), err=True)
+        return
+    try:
+        payload_decoded = token.split('.')[1]
+        # payload_decoded_correct_padding - is the same string with additional '=' symbols to make string length
+        # be dividable by 4:
+        payload_decoded_correct_padding = payload_decoded + b'=' * (4 - len(payload_decoded) % 4)
+        payload = json.loads(base64.b64decode(payload_decoded_correct_padding))
```

Comment:
`jwt` library is already included into the project (take a look `src/utilitities/storage/common.py` -> `StorageOperations.get_user()`

Reply:
@mzueva fixed via e74e38ae60622e3f862f2084f8e6a49145263748